nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -96,6 +96,10 @@ After going through this checklist, you're ready to go!
 	if(start_emagged)
 		obj_flags ^= EMAGGED
 
+/obj/structure/overmap/fighter/ComponentInitialize()
+	. = ..()
+	AddComponent(/datum/component/grav_taker) //ships are affected by gravity

```

Comment:
Should this not be on the base overmap class?

Reply:
so the idea is to not force it on every overmap ship, only things that need it. I was thinking things like asteroids and such would react badly to gravity, especially since they're unpowered. AI also at the moment haven't been tested, so that needs to happen before this all goes through

Reply:
Could have a variable on it is affected by gravity and have it false on asteroids